### ISSUE TITLE
The quick start for MMM does not work

### ISSUE THREAD

**Issue by HananInfiniGrow** [pymc-labs/pymc-marketing#357]
Running the example code does not work.

```
import pandas as pd
from pymc_marketing import mmm


data_url = "https://raw.githubusercontent.com/pymc-labs/pymc-marketing/main/datasets/mmm_example.csv"
data = pd.read_csv(data_url, parse_dates=['date_week'])

model = mmm.DelayedSaturatedMMM(
    data=data,
    target_column="y",
    date_column="date_week",
    channel_columns=["x1", "x2"],
    control_columns=[
        "event_1",
        "event_2",
        "t",
        "sin_order_1",
        "cos_order_1",
        "sin_order_2",
        "cos_order_2",
    ],
    adstock_max_lag=8,
)

model.fit()
model.plot_components_contributions();
```

produces the error:

> 
> ---------------------------------------------------------------------------
> TypeError                                 Traceback (most recent call last)
> Cell In[79], line 25
>       6 data = pd.read_csv(data_url, parse_dates=['date_week'])
>       8 model = mmm.DelayedSaturatedMMM(
>       9     data=data,
>      10     target_column="y",
>    (...)
>      22     adstock_max_lag=8,
>      23 )
> ---> 25 model.fit()
>      26 model.plot_components_contributions();
> 
> TypeError: ModelBuilder.fit() missing 1 required positional argument: 'X'

**Comment by twiecki** [comment]
Thanks for the report @HananInfiniGrow, would you like to submit a fix?

**Comment by michaelraczycki** [comment]
This looks like versioning issue, as the mmm_example notebook is already updated. Which version you can see in the left top corner? It might be you're looking at legacy documentation, in that case you should use older version of pymc-marketing. 
In the official example notebook I can see:
![Uploading Screenshot 2023-08-18 at 08.27.05.png…]()

**Comment by HananInfiniGrow** [comment]
It is in the latest official docs at https://www.pymc-marketing.io/en/stable/

p.s. thank you for publishing this open source. I know how much of a headache is it to implement this model alone. Please also support hierarchial and budget allocation optimization.

**Comment by michaelraczycki** [comment]
I'm sorry, I'm afraid I can't help in this case, the link you're providing shows me the doc version I've shared with you on the screenshot above. In general, we've altered the API a bit, to follow sklearn API. So now you need to divide your dataset into the X data frame, and y series (containing the target variable) and pass it along to `fit()` method. That should solve your problem. As to the example notebook, @ricardoV94 any ideas why we're seeing different documentation versions?

**Comment by HananInfiniGrow** [comment]
https://www.pymc-marketing.io/en/stable/
![Screenshot_2023-08-18-09-42-45-69_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/pymc-labs/pymc-marketing/assets/129379171/a44e51d6-c281-4e64-b191-dce19aee09a9)

**Comment by ricardoV94** [comment]
The documentation was only updated after we released the last version (in #340), and we haven't re-released so it doesn't show it up to date yet. 

If you go to `latest` you can see it already is corrected: https://www.pymc-marketing.io/en/latest/

We should be releasing later today, so everything will be in sync soon enough. Apologies for the confusion @HananInfiniGrow
